sdk/common/build.gradle.kts

```diff
@@ -25,13 +25,6 @@ dependencies {
   testImplementation("com.google.guava:guava-testlib")
 }
 
-testing {
-  suites {
-    register<JvmTestSuite>("testResourceDisabledByProperty") {}
-    register<JvmTestSuite>("testResourceDisabledByEnv") {}
-  }
-}
```

Comment:
These test suites are remnants from when the tests lived here instead of in autoconfigure.